README.md

```diff
@@ -10,26 +10,28 @@ Stop worrying about `-ldflags` and **`go get github.com/ahmetalpbalkan/govvv`**
 
 | Variable | Description | Example |
 |----------|-------------|---------|
-| **`main.GitCommit`** | short commit hash of source tree | `0b5ed7a` |
-| **`main.GitBranch`** | current branch name the code is built off | `master` |
-| **`main.GitState`** | whether there are uncommitted changes | `clean` or `dirty` | 
-| **`main.GitSummary`** | output of `git describe --tags --dirty --always` | `v1.0.0`, <br/>`v1.0.1-5-g585c78f-dirty`, <br/> `fbd157c` |
-| **`main.BuildDate`** | RFC3339 formatted UTC date | `2016-08-04T18:07:54Z` |
-| **`main.Version`** | contents of `./VERSION` file, if exists | `2.0.0` |
+| **`<pkg>.GitCommit`** | short commit hash of source tree | `0b5ed7a` |
```

Comment:
Let's not change these. The common case is `main` and this is kind of the marketing page for the project. Keeping this listing as `main.Xxx` helps people understand the project easier.